package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@edsc/geo-utils",
+  "version": "1.0.0",
+  "description": "Geographical utilities used by Earthdata Search",
```

Comment:
But this is also now used by smart hand-offs right? Thinking we should strip out these references to Earthdata Search expect for the fact that it was pulled from that codebase.. maybe?